### ISSUE TITLE
Tibetan text new numbering

### ISSUE THREAD

**Issue by ayya-vimala** [ayya-vimala/pali-networks#90]
New numbered Tibetan text and new menu will be prepared.

**Comment by sebastian-nehrdich** [comment]
I have just uploaded the new menus for Tibetan: https://github.com/ayya-vimala/pali-networks/blob/master/data-incoming/tibetan/tibetan_files.tsv
I might go ahead and replace the JSON/JS-menu that we have currently with the new data because I would like to send O the updated version of the Tibetan data today. So don't be surprised if I will make a change to the JS-code for the new menu!

**Comment by sebastian-nehrdich** [comment]
I have now also updated https://github.com/ayya-vimala/pali-networks/blob/master/src/suttalists/tibetan-menu.js.
Will push it together with the new Tibetan data to buddhist-db in about an hour.

**Comment by ayya-vimala** [comment]
Okay, I got it. But now there are a lot of files missing that are in the menu. Is that correct?
FI T06TD4021E (the one I always use for testing) is not there any more.

**Comment by sebastian-nehrdich** [comment]
Yes my upload got stuck halfway through. New files coming!

**Comment by ayya-vimala** [comment]
Upload still stuck then?

What are those files for that are marked `skt_`. The system cannot read it and as it seems tibetan and not sanskrit, it seems a bit weird. I am not in favor of having multilingual files because that way you have to make a lot of decisions on for instance texts that are partly translated and what is the difference between "translation" and "full parallel text"? Many of the early pali texts are translated into all other languages. At SC we regard them as full parallels.

Interesting first sentences btw. 
`        "segment": "+ ༣@@ ༣@@ q@@ ་@@ ལ@@ <unk> ི@@ ་@@ <unk> <unk> <unk> <unk> ོ@@ <unk> ི@@ ་@@ <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> ི@@ <unk> grags <unk> ི@@ ་@@ <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk> <unk>",
`
Is that supposed to be Vulan (like this Metta Sutta https://suttacentral.net/kp9/vu/spock) or Klingon https://suttacentral.net/kp9/kln/worf) :-)

**Comment by sebastian-nehrdich** [comment]
The problem ist that I still have problems with the precision of the data
because we changed the layout of the Tibetan. But I am now full-time
working on fixing the issues.
Probably I can upload the remaining files tomorrow at some point. And don't
worry about the skt_files, I uploaded them accidentally. These are actually
Sanskrit-texts which where machine-translated into Tibetan in order to find
matching passages for them, that's why the Klingon is there at times... ;)
